src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -101,12 +101,12 @@ def __init__(
         super().__init__(agg_key_funcs=agg_key_funcs, agg_default_func=agg_default_func)
         self._save_dir = save_dir
         self._name = name or ""
-        self._version = version
         self._sub_dir = sub_dir
         self._log_graph = log_graph
         self._default_hp_metric = default_hp_metric
         self._prefix = prefix
         self._fs = get_filesystem(save_dir)
+        self._version = version if version is not None else self._get_next_version()
```

Comment:
We shouldn't have any changes in tensorboard logger. Could you remove them?

Reply:
See https://github.com/Lightning-AI/lightning/pull/12292#discussion_r959484188